src/main/java/com/darkona/adventurebackpack/util/TConstructTab.java

```diff
@@ -0,0 +1,132 @@
+package com.darkona.adventurebackpack.util;
+
+import java.util.List;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.entity.EntityClientPlayerMP;
+import net.minecraft.client.renderer.RenderHelper;
+import net.minecraft.client.renderer.entity.RenderItem;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+
+import org.lwjgl.opengl.GL11;
+import org.lwjgl.opengl.GL12;
+
+import com.darkona.adventurebackpack.init.ModNetwork;
+import com.darkona.adventurebackpack.network.GUIPacket;
+import com.darkona.adventurebackpack.reference.BackpackTypes;
+import com.darkona.adventurebackpack.reference.LoadedMods;
+
+import cpw.mods.fml.common.Optional;
+import tconstruct.client.tabs.AbstractTab;
+import tconstruct.client.tabs.TabRegistry;
+
+public class TConstructTab {
+
+    private static boolean enabled;
+
+    static {
+        enabled = LoadedMods.TCONSTRUCT;
+    }
+
+    public static void registerTab() {
+        if (enabled) {
+            registerTabImpl(new Tab());
+        }
+    }
+
+    @Optional.Method(modid = "TConstruct")
```

Comment:
I don't think this method needs to be removed at runtime since `AbstractTab` / `TabRegistry` only need to exist if any method in this class is called, which isn't the case unless TiCon is loaded.